modular_skyrat/modules/undersized/undersized_quirk.dm

```diff
@@ -0,0 +1,40 @@
+#define UNDERSIZED_SPEED_SLOWDOWN 0.5
+
+/datum/quirk/undersized
+	name = "Undersized"
+	desc = "You, for whatever reason, are FAR too small, and will encounter some rough situations because of it."
+	gain_text = span_notice("That table looks large...")
+	lose_text = span_notice("Is it still the same size...?") //hehe
+	medical_record_text = "Patient is abnormally small."
+	value = 0
+	mob_trait = TRAIT_UNDERSIZED
+	icon = "minimize"
+	veteran_only = TRUE
+
+/datum/quirk/undersized/add()
+	var/mob/living/carbon/human/human_holder = quirk_holder
+	human_holder.dna.features["body_size"] = 0.3
```

Comment:
This will make players a bunch of unidentifiable pixels.
Just because servers like Hyper can get away with it doesn't mean we can in a server that has a ton more combat and antagonistic action on average. Even if the people using it are on a whitelist.

Reply:
I thought that we could trust people to handle it here, and it wouldn't be a real reflection of characters on other servers if it was too large but sure. I'll just make it .5 for the moment.